services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -48,6 +47,22 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar w
       }
     }
 
+    "updateProjectVersion" should {
+      "return correct version of project" taggedAs Service in {
+        val projectId = ProjectId("projectId")
+        val userId = UserId.random
+        val version = PipelineVersion("v1.0.0")
+
+        when(projectRepository.getProjectById(projectId)).thenReturn(Future(Some(dummyProject.copy(version = version))))
```

Comment:
```suggestion
        when(projectRepository.getProjectById(projectId)).thenReturn(Future.successful(Some(dummyProject.copy(version = version))))
```

Reply:
Use `Future.successful` for fast operations that don't require asynchronous execution
https://stackoverflow.com/questions/21466652/what-is-the-difference-between-future-successfulnone-and-futurenone